src/Blobify/Services/Storage/TokenService.cs

```diff
@@ -97,7 +97,7 @@ Uri url
         HttpContent? content = null
         )
     {
-        var httpClient = GetBearerTokenHttpClient(
+        using var httpClient = GetBearerTokenHttpClient(
```

Comment:
Isn't lifecycle handled by the factory?

```csharp
var bearerHttpClient = httpClientFactory.CreateClient(name);
```
https://github.com/devlead/Blobify/blob/cd5f4b9c3a4bcc2f69e0033577d31a1a6226833a/src/Blobify/Services/Storage/TokenService.cs#L42C10-L42C69

Reply:
fair enough :)